test/e2e-test/rule-test/util/test-rule-functions.spec.js

```diff
@@ -240,7 +240,7 @@ describe('testRule', function () {
         const reqJson = null;
         const resJson = JSON.parse(fs.readFileSync(path.join(__dirname, '../test-samples/FHIR-R4/sample3-right.json')));
         assert.ok(testRules.officialValidator(reqJson, resJson).valid);
-    }).timeout(60000);
+    }).timeout(180 * 1000);
```

Comment:
Does the issue caused by the timeout settings?

Reply:
Yes, the performance of github action server is not that stable, so the time usage of validating phase will fluctuate.
I change the timeout limit to a relatively big number to eliminate the influence of performance fluctuation.

Reply:
Got it. Thanks!